cli/pkg/kctrl/cmd/package/installed/delete.go

```diff
@@ -75,6 +75,7 @@ func NewDeleteCmd(o *DeleteOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 		AllowDisableWait: false,
 		DefaultInterval:  1 * time.Second,
 		DefaultTimeout:   5 * time.Minute,
+		DisableByDefault: o.pkgCmdTreeOpts.DisableWaitByDefault,
```

Comment:
This does conflict with `AllowDisableWait` in case of package installed delete 🤔

Reply:
Also, I think we need to add it to package repository add_or_update, kick and delete?

Reply:
Yeah I think we need it on kick and delete as well. Let me think about cases that it might run into with `AllowDisableWait`. We need to fallback to false and not allow a change. 
Should be noted that we will ONLY use this value in cases where we are not creating RBAC resources and a default SA is provided. I will try and take note of that someplace

Reply:
Changed it to `UnsetByDefault`.
Behaviour being: The older bool decides whether or not the flag is attached at all and the newer one decides the default value.
However, the older flag takes effect only if the newer one is unset.